tests/users/test_api_login.py

```diff
@@ -43,6 +46,29 @@ def setUp(self):
         db.session.add(self.first_user)
         db.session.add(self.second_user)
         db.session.commit()
+    def test_user_login_invalid_credentials(self):
```

Comment:
```suggestion

    def test_user_login_invalid_credentials(self):
```

Reply:
@PareenKarpe can you please do this? I think it would make the code more readable :)

Reply:
@isabelcosta Sure. I made those changes.